test/queue-test.js

```diff
@@ -460,12 +460,12 @@ describe('Queue', (it) => {
       // Override _waitForJob.
       const waitJob = queue._waitForJob, wait = helpers.deferred();
       let waitDone = wait.defer();
-      queue._waitForJob = function (...args) {
+      queue._waitForJob = function () {
         if (waitDone) {
           waitDone();
           waitDone = null;
         }
-        return waitJob.apply(this, args);
+        return waitJob.apply(this);
```

Comment:
`apply` like this just always feels weird to me :\
```suggestion
        return waitJob.call(this);
```

Reply:
You're right.